src/current/v23.1/troubleshooting-overview.md

```diff
@@ -20,6 +20,8 @@ If you experience an issue when using CockroachDB, try these steps to resolve th
 
 - [Replication Reports](query-replication-reports.html) let you query the status of your cluster's data replication, data placement, and zone constraint conformance.
 
+- The [CPU Profiler](cpu-profiler.html) lets you investigate CPU load by automatically capturing CPU profiles.
```

Comment:
Suggestion: Refer to this as the "Automatic CPU Profiler". 

As mentioned in another comment, the CPU profiles are geared for our CRL Support team to use to investigate erratic or spikey CPU load on a cluster. They're not designed with the end user in mind. So I recommend we tweak the language to not say "the automatic cpu profiler lets **you** investigate..." and more along the lines of it is another option to investigate these problems with CRL Support

Reply:
Actually on second thought, I'm not sure it makes sense to include this bullet in this page. Reading through this page, it sounds like actions that the user should take to troubleshoot issues. Where do escalations with CRL Support fit in? Is that outside the scope of this page? If so, let's remove this bullet. If not, then we should be more comprehensive here and bring up debug zips as well

Reply:
Made bullet point more comprehensive, added `cockroach debug zip` and `cockroach debug tsdump`.